api_app/analyzers_manager/file_analyzers/virushee.py

```diff
@@ -0,0 +1,106 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+import time
+from typing import Optional
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class Virushee(FileAnalyzer):
+    """Analyze a file against Virushee API"""
+
+    def set_params(self, params):
+        self.session = requests.Session()
+        api_key = self._secrets["api_key_name"]
+        if not api_key:
+            logger.info(
+                f"{self.__repr__()}, (md5: {self.md5}) -> Continuing w/o API key.."
+            )
+        else:
+            self.session.headers["Authorization"] = f"Bearer {api_key}"
```

Comment:
From virushee API docs, the header name is to be ` X-API-Key`

Reply:
Your changes are incomplete and also wrong (It doesn't seem like you tried running the code even once).  There are too many errors (using requests.get even though session was defined) and changes you forgot to make (in analyzer_config,json, env_file_ci). Please refrain from repeating such mistakes in the future if you want the maintainers to take your PR/work seriously.

EDIT: I pushed a commit fixing these errors.

Reply:
Not only this, you again pushed irrelevant changes. As I've said [before](https://github.com/honeynet/GreedyBear/pull/25#issuecomment-1011995786), do not push irrelevant changes because it makes the tests suite break.

Reply:
I've pushed commits fixing these errors.